server/api/products.js

```diff
@@ -11,3 +11,12 @@ router.get('/', async (req, res, next) => {
   }
 })
 
+router.post('/', async (req, res, next) => {
+  try {
+    const products = await Product.create(req.body)
+    res.json(products)
+  } catch (err) {
+    next(err)
+  }
+})
+
```

Comment:
You don't need to do it as a separate function, or break it down into json (the way Patrick did), but I do think you should destructure req.body before passing it to Product.create. This is a security issue, so that hackers can't sneak things into req.body. In other words, I think we should only pass to Product.create what it actually needs.